pkg/controller/datavolume-controller.go

```diff
@@ -84,9 +90,9 @@ func NewDataVolumeController(
 	// Add datavolume-controller types to the default Kubernetes Scheme so Events can be
 	// logged for datavolume-controller types.
 	cdischeme.AddToScheme(scheme.Scheme)
-	glog.V(Vdebug).Info("Creating event broadcaster")
+	glog.V(3).Info("Creating event broadcaster")
```

Comment:
What's the distinction between using `cdi.vDebug` above and hard coded ints in this file?

Reply:
It's not a major thing, but I figured we didn't really need a constant for 1, 2 and 3; and could manage to just use glog in a more normal manner so to speak.  The main motivation is I was working on elimintating pkg/common/common.go; there's one hold out left there but everything else sort of just "falls out" once you start poking at it.  This was the only thing really that needed to be in common by itself; seemed like it was more straight forward to just use glog normally like in K8s.